deepautoencoder/stacked_autoencoder.py

```diff
@@ -76,6 +82,13 @@ def fit(self, x):
                              batch_size=self.batch_size,
                              lr=self.lr, print_step=self.print_step)
 
+        if finetune:
```

Comment:
AFAIK, you conduct fine tuning after the whole network is trained and not after training of each layer.